CMakeLists.txt

```diff
@@ -55,15 +55,9 @@ if((main_project OR BUILD_TESTING_${PROJECT_NAME}) AND BUILD_TESTING)
 endif()
 
 # documentation
-if(main_project)
-    set(doc doc)
-else()
-    set(doc ${NAMESPACE}doc)
```

Comment:
The namespace was added in case a superproject also wants to define a target doc, so the simplest and best fix in my opinion would be to replace just the colons (e.g. instead of ${NAMESPACE} use ${PROJECT_NAME}_ )

Reply:
Done. I usually prefer to have unconditionally namespaced targets (with underscores) for this reason, but I don't want to breaking existing projects